test/dlc_tests/ecs/pytorch/training/test_ecs_pytorch_training.py

```diff
@@ -71,6 +81,9 @@ def test_ecs_pytorch_s3_plugin_training_cpu(cpu_only, ecs_container_instance, py
     Given above parameters, registers a task with family named after this test, runs the task, and waits for
     the task to be stopped before doing teardown operations of instance and cluster.
     """
+    _, image_framework_version = get_framework_and_version_from_tag(pytorch_training)
+    validate_or_skip_s3_plugin(image_framework_version)
```

Comment:
The problem with this approach is that the EC2 and ECS tests create their resources (and take the corresponding extra setup time) based on the use of the fixtures `ecs_container_instance` and `ecs_cluster_name` at the invocation of this test function.

Skipping the test using the `pt17_and_above_only` fixture in test inputs will skip this test before those resources are created, rather than afterwards, because this function is never actually invoked.

Therefore, could you put `pt17_and_above_only` in the test function inputs?